servicing.pyi

```diff
@@ -54,7 +54,7 @@ class Dispatcher:
         :param name: the name of the service
         """
 
-    def up(self, name: str) -> None:
+    def up(self, name: str, force: Optional[bool] = None) -> None:
```

Comment:
I would use a different name. Force is clearly used to skip the user prompt. Not forcing and skipping validation.

Reply:
you used force for `down` so i did the same here since they both bring up prompts but i agree i like `skip_prompt`